### ISSUE TITLE
Drones Seem Broken in latest 1140 Build

### ISSUE THREAD

**Issue by PRAWKC** [Bara20/TroubleinTerroristTown#150]
When I buy a drone or camera as Detective or T does not seems to work. The Ta Grenade just explodes then says you have no drones.

**Comment by Bara20** [comment]
@Keplyx

**Comment by Keplyx** [comment]
@PRAWKC are you using the [latest version](https://github.com/Keplyx/cameras-and-drones/raw/master/plugins/cameras-and-drones.smx) of Cameras and Drones? If so, could you try to buy/use drones and cameras without the TTT plugin and see if it works? I don't have any problems on my test server. And last, do you have any other plugin that might break mine?

@Bara20 to buy cameras and drones in your shop, you use the native I gave you right? You don't simply give a TA-Grenade to the player? Because if you enable the use of TA-Grenades with the cvar, the only way to use cameras and drones is to buy one. 

I don't really know why it doesn't work for you, because Bara said it worked in his test server.

**Comment by PRAWKC** [comment]
@Keplyx I have the latest version. I also can buy the drones and cameras with cd_buy but when I try using the Traitor/Detective Menu it seems to only give me a normal Tactical Awareness Grenade on latest build 1143. No I do not have other plugins that would break it because in a previous version of TTT it was working fine.

**Comment by Bara20** [comment]
@Keplyx https://github.com/Bara20/TroubleinTerroristTown/blob/master/addons/sourcemod/scripting/ttt/ttt_dronescameras.sp#L84-L115
That's the way.

**Comment by Bara20** [comment]
It works when cd_usetagrenade 0. But... if it 0 then doesn't work ta-grenades.

**Comment by Keplyx** [comment]
@PRAWKC THanks for the detailed anwser, it help a lot!

@Bara20 I think the problem is you don't use the "BuyPlayerGear" native I made. If I understood your code correctly, you set the player gear, then give a tagrenade to the player. But the cvar "cd_usetagrenade" I added is supposed make the player able to use tagrenades as long as he doesn't buy a camera/drone. So here the player never buy the camera/drone with the TTT menu, but is only given a classic tagrenade.
